monkey/infection_monkey/network/network_scanner.py

```diff
@@ -119,5 +138,18 @@ def _is_any_ip_in_subnet(ip_addresses, subnet_str):
                 return True
         return False
 
+    def scan_machine(self, victim):
+        """
+        Scans specific machine using given scanner
```

Comment:
This documentation grew outdated; It's not using a given scanner, but rather the instances' scanners.